src/Agent.Worker/TaskRunner.cs

```diff
@@ -190,7 +190,7 @@ public async Task RunAsync()
                     string key = input?.Name?.Trim() ?? string.Empty;
                     if (!string.IsNullOrEmpty(key))
                     {
-                        inputs[key] = input.DefaultValue?.Trim() ?? string.Empty;
+                        inputs[key] = input.DefaultValue ?? string.Empty;
```

Comment:
Can we add tests?

Is this going to break anyone that isn't currently broken? I guess anyone who has spacing now on accident will start to break. Will it be clear to them why it's broken?

Reply:
> Is this going to break anyone that isn't currently broken? I guess anyone who has spacing now on accident will start to break. Will it be clear to them why it's broken?

To be a little more exact, anyone who has spacing and is relying on there not being spacing is going to break. It probably won't be immediately clear, I didn't have a great solution for that. In theory we could warn that they have trailing spaces for a while, it would probably flag a bunch of false positives though.

Reply:
I thought more about this - I think not trimming all whitespace scares me. I updated to still trim spaces, but this will leave linebreaks. It solves the ticket in question in what I think is a pretty safe way.

Because of how yaml is structured/the importance of whitespace, I think its not unreasonable to trim spaces.